### DISCUSSION TITLE
Create different test suite according to the different environments

### DISCUSSION THREAD

**Discussion by aaujayasena** [cypress-io/cypress#14775] [Questions and Help] [answered]
I have a requirement to execute test in different environments.  But the concern is according to the environment some of the test are invalid. Hence I have to disable the test according to the environment. I have update the cypress.json file with following parameters, but is there any other way that I can override these value at run time.  

![image](https://user-images.githubusercontent.com/39077751/105980608-13aa2280-60bb-11eb-9a0e-a52f82d76466.png)

I have tried the export command as below and doesn't work.
`export CYPRESS_testFiles=["**/e2e/*.spec.*", "**/scim/scim2.0/scim2.0-tests/**-consumer-local-user.spec.*"]`

**Comment by bahmutov** [comment, marked as answer]
You probably want to have either a separate `cypress.json` per environment with different `ignoreTestFiles` patterns, or skip tests based on the environment using the plugin https://github.com/cypress-io/cypress-skip-test

**Comment by dariuszteooh** [comment]
I am doing that with cypress/skip-test. You can get it here: https://www.npmjs.com/package/@cypress/skip-test
The way I'm doing it is that I've put environmentName field in my env files and then in the tests I want to ignore I have this: 
```
before(function () {
        onlyOn(Cypress.env('envName') != 'production')
    })
```
Meaning I only run this test on environments other than production.
